builds/service-onboarding-build-pack/Jenkinsfile

```diff
@@ -87,22 +87,22 @@ node  {
 						else if (service_config['runtime'].startsWith("go")) {
 							service_template = 'api-template-go'
 						}
-            else if(service_config['runtime'].startsWith("c#")){
-              if(service_config['platform'] == "azure"){
-                service_template = 'azure-api-template-csharp'
-              }
-            }
+						else if(service_config['runtime'].startsWith("c#")){
+							if(service_config['platform'] == "azure"){
+								service_template = 'azure-api-template-csharp'
+							}
+						}
 						break
 
 					case "function":
-            if (service_config['platform'] == "azure") {
-              if (service_config['runtime'].startsWith("nodejs")) {
-                service_template = "azure-function-template-nodejs"
-              }
-              else if (service_config['runtime'].startsWith("c#")) {
-                service_template = 'azure-function-template-csharp'
-              }
-            }
+						if (service_config['platform'] == "azure") {
+							if (service_config['runtime'].startsWith("nodejs")) {
+								service_template = "function-template-nodejs"
+							}
+							else if (service_config['runtime'].startsWith("c#")) {
+								service_template = 'function-template-csharp'
```

Comment:
@oscarbrooksdev9 I changed it to azure-function-template per Satish suggested.  Please dont override my work